frog/imports/client/Wiki/index.js

```diff
@@ -720,7 +898,7 @@ class WikiComp extends React.Component<WikiCompPropsT, WikiCompStateT> {
                 goToPage={this.goToPage}
                 dashboardSearch={this.state.dashboardSearch}
                 side={this.state.mode === 'splitview' ? 'left' : null}
-                disableEdit={this.props.embed || this.state.settings?.readOnly}
+                checkEdit={() => this.editAccess('editPage')}
```

Comment:
`'editPage'` does not appear to be a case checked by an if statement inside `checkEdit`

Reply:
It is implicitly being checked since there are only 2 possibilities `'editPage'` and `'createPage'`. If you insist, I could change the `if` statement to make this explicit

Reply:
Make it a switch statement and move code away to separate functions